vuepress/docs/v6.3.2/tutorials/samples/landing-page.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Solution Templates 
+
+## Overview
+
+The Entando Application Composition Platform offers four Solution Templates to demonstrate various business capabilities and integrations. Each Solution Template provisions the building blocks and structure scoped for a particular implementation. 
+
+Solution Templates are available in the [public Entando Hub](http://hubdev.okd-entando.org/entando-de-app/en/test.page#/) for installation in a user environment, where turnkey functionality can be customized and extended.
+
+TODO: change link (added as placeholder to pass workflow check)
+
+## Solution Template Details
+
+Specific features and use cases are highlighted below:
+
+- **Entando Customer Portal**:\
```

Comment:
I like the flow overall but I do think including a thumbnail would be useful. We could even just use the actual thumbnails I have on the Hub itself if you want. That would be more generic, still add a little flair to the page, and let us save the full screenshots for the detailed pages.